files/usr/bin/container-start.sh

```diff
@@ -108,6 +109,44 @@ fi
 
 cp /etc/resolv.conf $BASEDIR/$SCHEDID/resolv.conf.tmp
 
+### ENABLE NEAT PROXY #################################################
```

Comment:
Maybe, at this point as well as in container-stop, whether a neat experiment is requested or not, do a 
rm /etc/circle.d/60-*-neat-proxy.rules || true
to make sure no conflicting rules exist.

Reply:
Cleanup on start added in commit 0e750311dd9039cd80e3d1fafbb47086c57b43b9.
container.stop already have a cleanup of rules in https://github.com/MONROE-PROJECT/Scheduler/blob/0e750311dd9039cd80e3d1fafbb47086c57b43b9/files/usr/bin/container-stop.sh#L85